README.md

```diff
@@ -18,6 +33,9 @@ Consider the following goals of the onboarding document:
 + Living: The document is expected to change.
 + Local Resources: (should this be part of a separate document?)
 
+## Useful links
+* [Pividori Lab logos](https://github.com/pivlab/pivlab-website/tree/main/images/logo).
+
```

Comment:
we'll need a picture of the group when Haoyu joins the lab!

Reply:
Absolutely!